agent/app/agent_capability_unix.go

```diff
@@ -153,6 +153,10 @@ func (agent *ecsAgent) appendFirelensFluentbitCapabilities(capabilities []*ecs.A
 	return appendNameOnlyAttribute(capabilities, attributePrefix+capabilityFirelensFluentbit)
 }
 
+func (agent *ecsAgent) appendEFSCapabilities(capabilities []*ecs.Attribute) []*ecs.Attribute {
```

Comment:
needs to be there in `agent_capability_windows.go` as well

Reply:
I thought we are only supporting linux for now?

Reply:
oh yes! just a dummy method returning attributes.

Reply:
I see now, pushed